ui/userguide/UserGuideBody.jsx

```diff
@@ -76,14 +84,20 @@ const UserGuideBody = function UserGuideBody() {
             <div className="card-body">
               <table id="legend-other">
                 <tbody>
-                  {notations.map(({ classes, explanation, text }) => (
+                  {notations.map(({ classes, icon, explanation, text }) => (
                     <tr key={classes}>
-                      <th className="superseded">
+                      <th>
                         <button
                           type="button"
                           className={`btn ug-btn ${classes}`}
                         >
                           {text || 'Th'}
+                          {icon && (
```

Comment:
Not sure what's causing this, but I see an artifact after the stars:
<img width="295" alt="screenshot 2019-02-04 16 59 37" src="https://user-images.githubusercontent.com/419924/52247335-5d7e1a80-289e-11e9-9109-bd442312dc9c.png">

Reply:
> Not sure what's causing this, but I see an artifact after the stars:

Ah yes sorry this is due to the way job-view sets the font icon/content manually:
https://github.com/mozilla/treeherder/blob/405a8c1ced5c23a503f9ccf430661d57c5944e09/ui/css/treeherder-job-buttons.css#L175-L191

I didn't see this locally since I've removed that part in the WIP for jobs-view that I split out of this PR.

I've updated this PR to remove the userguide conversion - I think it will be best to change it at the same time as jobs view, given the shared CSS.